packages/nimble-components/src/banner/styles.ts

```diff
@@ -0,0 +1,214 @@
+        padding-right: 8px;
+        white-space: nowrap;
+    }
+
+    :host([title-hidden]) slot[name='title'] {
+        ${
+            /**
+             * Hide content visually while keeping it screen reader-accessible.
+             * Source: https://webaim.org/techniques/css/invisiblecontent/#techniques
+             * See discussion here: https://github.com/microsoft/fast/issues/5740#issuecomment-1068195035
+             */
+            ''
+        }
+        display: inline-block;
+        height: 1px;
+        width: 1px;
+        position: absolute;
+        margin: -1px;
+        clip: rect(1px, 1px, 1px, 1px);
+        clip-path: inset(50%);
+        overflow: hidden;
+        padding: 0;
+    }
+
+    .controls {
+        height: ${controlHeight};
+        margin-left: auto;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        align-self: flex-start;
+    }
+
+    slot[name='action'] {
+        display: flex;
+        align-content: center;
+        margin-left: ${standardPadding};
+        white-space: nowrap;
+    }
+
+    slot[name='action']::slotted(nimble-anchor) {
+        ${linkFontColor.cssCustomProperty}: ${White};
+        ${linkDisabledFontColor.cssCustomProperty}: ${White};
+        ${linkActiveFontColor.cssCustomProperty}: ${hexToRgbaCssColor(
+            White,
+            0.6
+        )};
+        font-size: 12.8px;
+    }
+
+    slot[name='action']::slotted(nimble-button) {
+        height: ${controlSlimHeight};
+        ${buttonLabelFontColor.cssCustomProperty}: ${White};
+        ${fillSelectedColor.cssCustomProperty}: ${hexToRgbaCssColor(
+            White,
+            0.2
+        )};
+        ${borderHoverColor.cssCustomProperty}: ${White};
+    }
+
+    slot[name='action']::slotted(nimble-button[appearance='outline']) {
+        ${actionRgbPartialColor.cssCustomProperty}: ${White}
+    }
+
+    .dismiss {
+        width: 48px;
+        display: flex;
+        justify-content: center;
+    }
+
+    .dismiss nimble-button {
+        height: 16px;
+        width: 16px;
+        ${iconSize.cssCustomProperty}: 14px;
+        ${buttonLabelFontColor.cssCustomProperty}: ${White};
+        ${borderHoverColor.cssCustomProperty}: transparent;
+        ${fillSelectedColor.cssCustomProperty}: ${hexToRgbaCssColor(
+            White,
+            0.2
+        )};
+    }
+
+    .dismiss nimble-button:focus-within {
+        outline: 2px solid ${White};
+    }
+
+    .dismiss nimble-button:hover {
+        background: ${hexToRgbaCssColor(White, 0.2)};
+    }
+`.withBehaviors(
+    themeBehavior(
+        Theme.light,
+        css`
+            :host {
+                background: ${Black75};
+            }
+
+            :host([severity='error']) {
+                background: ${Fail100LightUi};
+            }
+
+            :host([severity='warning']) {
+                background: ${Warning100LightUi};
+            }
+
+            :host([severity='info']) {
```

Comment:
guessing storybook will help capture these but make sure to update with new attribute values

Reply:
Yep, updated.